nextcloud_scan.py

```diff
@@ -66,42 +66,58 @@ def getResultJson(self):
         result = self.doScan()
 
         result['vulnerabilitiesCount'] = len(result['vulnerabilities'])
-        result['hardeningsMissing'] = sum( 1 for isHardened in result['hardenings'].values() if isHardened == False )
-        result['headersMissing'] = sum( 1 for headerPresent in result['setup']['headers'].values() if headerPresent == False )
+        result['hardeningsMissing'] = sum(1 for isHardened in result['hardenings'].values() if isHardened is False)
+        result['headersMissing'] = sum(1 for headerPresent in result['setup']['headers'].values() if headerPresent is False)
 
-        return json.dumps(result, separators=(',',':'))
+        return json.dumps(result, separators=(',', ':'))
+
+    def getFormattedResultJson(self):
+        result = self.doScan()
+
+        result['vulnerabilitiesCount'] = len(result['vulnerabilities'])
+        result['hardeningsMissing'] = sum(1 for isHardened in result['hardenings'].values() if isHardened is False)
+        result['headersMissing'] = sum(1 for headerPresent in result['setup']['headers'].values() if headerPresent is False)
+
+        # Hier erfolgt die formatierte Ausgabe mit Einrückungen
+        return json.dumps(result, indent=4, separators=(',', ':'))
 
 def main():
     parser = argparse.ArgumentParser(
-                                    description='Run a Nextcloud security scan at scan.nextcloud.org',
-                                    )
+        description='Run a Nextcloud security scan at scan.nextcloud.org',
+    )
```

Comment:
Please don't unnecessarily change code whitespace in a PR; it makes it harder to review the meaningful changes. Could you re-push without the indentation-only changes?